.gitignore

```diff
@@ -0,0 +1,6 @@
+
+*.pyc
+*.pyc
+website/__pycache__/interfacing.cpython-311.pyc
+website/__pycache__/managers.cpython-311.pyc
+website/__pycache__/views.cpython-311.pyc
```

Comment:
Make sure to add `**/__pycache__/` or something along those lines to your `.gitignore`. These folders are created automatically by the Python interpreter and only contain system-specific bytecode.